_layouts/report.md

```diff
@@ -19,7 +19,7 @@
         <br>
             <div class="row no-margin">
                 <div class="col m12">
-                    <h4 class="left light">Google Summer of Code 2017 Work Product Submission </h4>
+                    <h4 class="left light">Google Summer of Code 2018 Work Product Submission </h4>
```

Comment:
this would effect all of the 2017 reports also.

I've created https://github.com/coala/projects/issues/608